content/guide/ui/styling.md

```diff
@@ -0,0 +1,694 @@
+---
+title: Styling
+---
+
+In NativeScript you style the UI similarly to how you do it in a web application using Cascading Style Sheets (CSS), or by changing the style property of the views in JavaScript/Typescript. For CSS styling, NativeScript supports tools such as [Tailwind](https://docs.nativescript.org/plugins/tailwindcss.html), [SASS](#using-sass). 
+
+Similarly to the DOM Style Object, each View instance exposes a style property, which holds all the style properties for the view. When the view is displayed, all its style properties are applied to the underlying native widget.
+
+## Applying CSS styles
+
+In NativeScript the CSS styles can be set on 4 different levels:
+
+- [Application-wide CSS](#application-wide-css): Applies to every application page
+- [Page-specific CSS](#page-specific-css): Applies to the page's UI views
+<!-- - [Component-specific CSS](#component-specific-css): Applies for component only -->
+- [Inline CSS](#inline-css): Applies directly to a UI view
+- [Platform-specific CSS](#platform-specific-css)
+
+If there is CSS declared on different levels&mdash;all will be applied. The inline CSS will have the highest priority and the application CSS will have the lowest priority.
```

Comment:
```suggestion
If there is CSS declared on different levels&mdash;all will be applied. The inline CSS will have the highest priority and the application CSS will have the lowest priority.
```

If CSS is declared at different levels, all styles will be applied. Inline CSS takes the highest priority, while application-level CSS has the lowest priority.